single-solution/SolutionPage.js

```diff
@@ -1,63 +1,63 @@
-$(function(){
-   
-    var totalVolume = $("#totalVolume").clone();
-
-    /**
-     * 'Next' button control function. This function is called when the 'Next' button is pressed
-     */
-    $("#nextButton").click(function(){
```

Comment:
this built-in "nextButton" function was causing fillFields to be called twice, and caused even single solution - solid answer to show up as "undefined".